man/loader.conf.xml

```diff
@@ -106,9 +106,10 @@
         will be stored as an EFI variable in that case, overriding this option.
         </para>
 
-        <para>If the timeout is disabled, the default entry will be booted
-        immediately. The menu can be shown by pressing and holding a key before
-        systemd-boot is launched.</para>
+        <para>If set to <literal>0</literal> the timeout is disabled and the default
+        entry will be booted immediately. The menu can be shown by pressing and holding
+        a key before systemd-boot is launched. Setting this to <literal>-1</literal>
+        disables the timeout while always showing the menu.</para>
```

Comment:
special numbers like this are ugly as fuck. the efi var is a string anyway, hence just use a proper string to indicate this case, i.e. say you have to specify "never" or so here as special value.

And then internally map this to UINTN_MAX (i.e. the closest that comes to the concept of "infinity")